Ports/ja2/patches/0001-Make-it-build-with-SDL-2-instead-of-1.2.patch

```diff
@@ -1126,20 +1126,20 @@ index 263f9b1..c232813 100644
 --- a/sgp/Platform.h
 +++ b/sgp/Platform.h
 @@ -43,11 +43,9 @@
-   #define __func__ __FUNCTION__
- #endif
- 
--#if !defined(_WIN32)
--  /* Not Visual Studio, not MINGW */
--  #define __max(a, b) ((a) > (b) ? (a) : (b))
--  #define __min(a, b) ((a) < (b) ? (a) : (b))
--#endif
-+/* Not Visual Studio, not MINGW */
-+#define __max(a, b) ((a) > (b) ? (a) : (b))
-+#define __min(a, b) ((a) < (b) ? (a) : (b))
- 
- /**************************************************************
-  *
+   #define __func__ __FUNCTION__
+ #endif
+ 
+-#if !defined(_WIN32)
+-  /* Not Visual Studio, not MINGW */
+-  #define __max(a, b) ((a) > (b) ? (a) : (b))
+-  #define __min(a, b) ((a) < (b) ? (a) : (b))
+-#endif
++/* Not Visual Studio, not MINGW */
++#define __max(a, b) ((a) > (b) ? (a) : (b))
++#define __min(a, b) ((a) < (b) ? (a) : (b))
```

Comment:
This patch is uh... very suspicious.  2.5k lines?  This change in particular I don't know why it was updated by your process here, but... idk. I think this port's update should be reverted and this patch should be seriously reviewed some other time.